src/groups/special_euclidean_group.jl

```diff
@@ -777,6 +777,17 @@ function LinearAlgebra.norm(
     return norm([n1, n2])
 end
 
+function lie_bracket!(
+    ::LieAlgebra{ℝ,<:SpecialEuclideanGroupOperation,<:SpecialEuclideanGroup},
+    Z::AbstractMatrix,
+    X::AbstractMatrix,
+    Y::AbstractMatrix,
+)
+    mul!(Z, X, Y)
+    mul!(Z, Y, X, -1, true)
+    return Z
+end
```

Comment:
If I include `SpecialEuclideanMatrixTangentVector` then `mul!` fails.
So, I don't know what the preferred way to deal with `SpecialEuclideanMatrixTangentVector` is here.

Reply:
I extended `Base.:*` for `SpecialEuclideanMatrixTangentVector` to see if it now works.

Reply:
For that we should check with @mateuszbaran because I think that would allocate too much and I would prefer to still use `mul!` for them as well, but you could also extend `Base.mul!` here then.

Reply:
You are right, but this uncarefulness we also have for the “normal” matrix variant for now – for historical reasons, since the old one had that as well. But we should maybe be more careful there then.